packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -57,12 +71,13 @@ export class MeasureToolsUiItemsProvider implements UiItemsProvider {
         return [
           ToolbarItemUtilities.createGroupButton(
             "measure-tools-toolbar",
-            500,
+            this._props.itemPriority!,
```

Comment:
I am not a fan of adding the `!`

Reply:
But itemPriority is a required field of createGroupButton, so if I remove the exclamation mark TypeScript complains of `Argument of type 'number | undefined' is not assignable to parameter of type 'number'.` even though this._props.itemPriority is never undefined because it is always defaulted to either 500 or the user's custom input. Do you think there's a better way to do this?

Reply:
instead of providing defaults in the consturctor you could just add the defaults here